lib/emerald/PreProcessor.rb

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env ruby
```

Comment:
Was there anything changed to this file, or does git just not recognize the file has been moved?